detekt/detekt-config.yml

```diff
@@ -85,7 +85,7 @@ complexity:
     threshold: 10
     includeStaticDeclarations: false
     includePrivateDeclarations: false
-  ComplexMethod:
+  CyclomaticComplexMethod:
```

Comment:
A lot of lint rules like this one were either deprecated or migrated to the `libraries` ruleset.